### PR TITLE
Implemented a unit test for FindOffers page

### LINKED ISSUE #98: (SP: 1) Write unit test for "Find offers" page
### Page unit test
Unit test for "**Find offers**" page
Scenaries descriptions:
- [x] It should render page
- [x] Text of the page must be in the document

[Link to component](https://github.com/Space2Study-UA-1125/frontEnd/blob/develop/src/pages/find-offers/FindOffers.jsx)

Current coverage:
Image

### PR DESCRIPTION
**Pull request by vldkk**
image
image

### CODE REVIEW

**Review by VictoriaRodik — APPROVED** [review]

**Review by holus-bolus — APPROVED** [review]

**Review by StasRusnak1 — APPROVED** [review]

**Review by OliaHrytsak — APPROVED** [review]

File: tests/unit/pages/find-offers/FindOffers.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { screen } from '@testing-library/react'
+import { renderWithProviders } from '~tests/test-utils'
+import { it } from 'vitest'
```
Comment (vlad-khrychov) [inline tests/unit/pages/find-offers/FindOffers.spec.jsx]:
```suggestion
```

File: tests/unit/pages/find-offers/FindOffers.spec.jsx
Comment (vlad-khrychov) [inline tests/unit/pages/find-offers/FindOffers.spec.jsx:1]: We need to mock `PageWrapper` component.

**Review by vlad-khrychov — CHANGES_REQUESTED** [review]
In issue we have 2 items, so at least 2 tests must be available

File: tests/unit/pages/find-offers/FindOffers.spec.jsx (hunk 1 above)
Comment (vldkk) [reply tests/unit/pages/find-offers/FindOffers.spec.jsx]: Changed it, check, please.

File: tests/unit/pages/find-offers/FindOffers.spec.jsx
Comment (vldkk) [reply tests/unit/pages/find-offers/FindOffers.spec.jsx:1]: Sure) Added a mock, could you check, please?

**Review by vlad-khrychov — APPROVED** [review]

**Pull request merged**